guides/hosting/performance/cart-storage.md

```diff
@@ -16,11 +16,33 @@ Redis is better suited in high-throughput scenarios, therefore you should use Re
 
 To use Redis, create a `config/packages/shopware.yml` file with the following content:
 
+<Tabs>
+<Tab title="Before v.6.6.7">
```

Comment:
I think we need to exchange the versions numbers with 6.6.8, as 6.6.7 will be released this or next week

Reply:
Updated